common/src/View/CameraTool2D.cpp

```diff
@@ -36,7 +36,7 @@ namespace View
 static float adjustSpeedToZoom(
   const Renderer::OrthographicCamera& camera, const float speed)
 {
-  return speed * vm::min(1.0f, 1.0f / camera.zoom());
+  return speed / vm::max(0.001f, camera.zoom());
```

Comment:
I don't understand this change, could you explain this please?

Reply:
I had originally copied over and adapted the code from `CameraTool3D`, where it was:

```cpp
return speed * vm::min(1.0f, camera.zoomedFov() / camera.fov());
```

The ratio of the zoom / original FoV produces a value that's > 1 when zoomed out and < 1 when zoomed in. So when zoomed in we want the pan to be slower, and when zoomed out to be faster.

After fiddling with it for a bit I realized that this was backwards for the 2D camera, since the `camera.zoom()` value will be < 1 when zoomed out and > 1 when zoomed in, so I switched the `*` to a `/` to invert it. The `vm::max()` is there simply as a safeguard to prevent producing Infinity / NaN values if the zoom value is ever 0 (which shouldn't happen)